API/Controllers/SchoolsController.cs

```diff
@@ -0,0 +1,132 @@
+
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using API.Models;
+using API.Data;
+using API.DTOs;
+
+namespace API.Controllers;
+
+[ApiController]
+[Route("/[controller]")]
+public class SchoolsController : ControllerBase
+{
+    private readonly myDbContext m_DbContext;
+    
+    public SchoolsController(myDbContext i_DbContext) 
+    {
+        m_DbContext = i_DbContext;
+    }
+    
+    [HttpGet]
+    public async Task<ActionResult<string>> GetSchools()
```

Comment:
I liked that you wrote async code by default 
just please add a cancellation token with default value
meaning:     
public async Task<ActionResult<string>> GetSchools(CancellationToken cancellationToken = default)

var schools = await _MyDbContext.Schools.ToListAsync(cancellationToken : cancellationToken);

Reply:
Fixed